crates/oxc_transformer/src/common/helper_loader.rs

```diff
@@ -193,6 +195,8 @@ impl Helper {
             Self::ReadOnlyError => "readOnlyError",
             Self::WriteOnlyError => "writeOnlyError",
             Self::CheckInRHS => "checkInRHS",
+            Self::Decorate => "decorate",
+            Self::DecorateParam => "decorateParam",
```

Comment:
Open to suggestions for the naming, in TypeScript, it calls `decorate` and `param`, but I think `param` is not intuitive, so I renamed it to `decorateParam`

Reply:
That naming sounds good to me!